### PR TITLE
Fixed quest "The Power of Light" ( http://www.wowhead.com/quest=26918 ) ...

### PR DESCRIPTION
**Pull request by Basti420**
Fixed quest "The Power of Light" ( http://www.wowhead.com/quest=26918 )Seal of Command hit now counts
Fixed quest "Report to Goldshire" ( http://www.wowwiki.com/Quest:Report_to_Goldshire ) requires level 3 to start
Fixed startup error which was caused by 2014_03_05_00_world_locales_quest.sql in a fresh db install

### CODE REVIEW

File: src/server/scripts/EasternKingdoms/elwynn_forest.cpp [hunk 1]
```diff
@@ -208,6 +209,7 @@ class npc_training_dummy_start_zones : public CreatureScript
                 case SPELL_CORRUPTION_2:
                 case SPELL_CORRUPTION_3:
                 case SPELL_PAUME_TIGRE:
+				case SPELL_SEAL_OF_COMMAND:
```
Comment (FALL1N1) [inline src/server/scripts/EasternKingdoms/elwynn_forest.cpp]: tabs

Comment (Basti420) [inline src/server/scripts/EasternKingdoms/elwynn_forest.cpp]: yes you're right

File: src/server/game/Globals/ObjectMgr.cpp [hunk 2]
```diff
@@ -4297,7 +4297,7 @@ void ObjectMgr::LoadQuestLocales()
 
     _questLocaleStore.clear();                                // need for reload case
 
-    QueryResult result = WorldDatabase.Query("SELECT entry, "
+    QueryResult result = WorldDatabase.Query("SELECT Id, "
```
Comment (Kittnz) [inline src/server/game/Globals/ObjectMgr.cpp:4300]: Didn't we change this before?

Comment (Basti420) [inline src/server/game/Globals/ObjectMgr.cpp:4300]: in the database yes but not in the objectmgr got a error every time i do a new fresh database import

edit:

https://github.com/MistCore/MistCore/commit/f7a635d690bdbbe72cccfb8638ecd49401267da0 should have fixed it but with a fresh pull from the MistCore Repo there is still 

QueryResult result = WorldDatabase.Query("SELECT entry, "

and not
QueryResult result = WorldDatabase.Query("SELECT Id, "

edit2:
fix was removed with this commit https://github.com/MistCore/MistCore/commit/ad763c0f2bacb60876dc708274468f7392307d99

**Comment by Kittnz** [comment]
Fix the tabs and you have a merge

**Comment by Basti420** [comment]
should be correct now

**Pull request merged**